### PR TITLE
use go replace

### PR DESCRIPTION
**Pull request by kevindiu**

### CODE REVIEW

**Comment by WindzCUHK** [comment]
@kevindiu related?
https://github.com/yahoojapan/garm/pull/39/commits/0c11dae341665659c36a45abb9b0496b55a5c0a1
https://github.com/yahoojapan/garm/pull/39

File: go.mod [hunk 1]
```diff
@@ -2,12 +2,19 @@ module github.com/yahoojapan/garm
 
 go 1.13
 
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
```
Comment (kevindiu) [inline go.mod:6]:
```suggestion
	k8s.io/api => k8s.io/api v0.18.3
```

File: go.mod [hunk 2]
```diff
@@ -2,12 +2,19 @@ module github.com/yahoojapan/garm
...
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190918201827-3de75813f604
```
Comment (kevindiu) [inline go.mod:7]:
```suggestion
	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.3
```

File: go.mod [hunk 3]
```diff
@@ -2,12 +2,19 @@ module github.com/yahoojapan/garm
...
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
```
Comment (kevindiu) [inline go.mod:8]:
```suggestion
	k8s.io/apimachinery => k8s.io/apimachinery v0.18.3
```

File: go.mod [hunk 4]
```diff
@@ -2,12 +2,19 @@ module github.com/yahoojapan/garm
...
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```
Comment (kevindiu) [inline go.mod:9]:
```suggestion
	k8s.io/client-go => k8s.io/client-go v0.18.3
```

**Comment by kevindiu** [comment]
yes seems they are related. we need to use go replace to replace k8s packages version because they manage k8s library version badly :( .....

**Comment by WindzCUHK** [comment]
> yes seems they are related. we need to use go replace to replace k8s packages version because they manage k8s library version badly :( .....

Thx, will handle this issue in #39.

**Pull request closed without merging**